lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -386,18 +386,6 @@ describe('GatherRunner', function() {
       ));
   });
 
-  it('returns an empty object if a falsey value is passed in to extraHeaders', () => {
-    const driver = getMockedEmulationDriver(null, null, null, null, params => params.headers);
-
-    return GatherRunner.beforePass({
-      driver,
-      flags: {
-        extraHeaders: undefined,
-      },
-      config: {gatherers: []},
-    }).then((returnValue) => assert.deepStrictEqual(returnValue, {}));
-  });
```

Comment:
This was a strange test relying on an empty `{}` returned from `GatherRunner.beforePass` when given no gatherers and an `undefined` headers object. `Driver. setExtraHTTPHeaders` no longer returns a `{}` in that case, and a test for normal operation when given an `undefined` headers object is found two tests above this, so I've just removed it.